src/commands/plan.yml

```
@@ -95,15 +97,16 @@ steps:
         fi
         if [[ -n "<< parameters.var_file >>" ]]; then
         for file in $(echo "<< parameters.var_file >>" | tr ',' '\n'); do
-            if [[ -f "$file" ]]; then
+            if [[ -f "$module_path/$file" ]]; then
                 PLAN_ARGS="$PLAN_ARGS -var-file=$file"
-            elif [[ -f "$module_path/$file" ]]; then
-                PLAN_ARGS="$PLAN_ARGS -var-file=$module_path/$file"
             else
                 echo "var file '$file' wasn't found" >&2
                 exit 1
             fi
         done
         fi
         export PLAN_ARGS
-        terraform plan -input=false -no-color -out=plan.out $PLAN_ARGS "$module_path"
+        if [[ -n "<< parameters.plan_file >>" ]]; then
+          export plan_file="-out=<< parameters.plan_file >>"
+        fi
+        terraform -chdir="$module_path" plan -input=false -no-color $plan_file $PLAN_ARGS
```

We could leverage the existing `PLAN_ARGS` variable:
```suggestion
        if [[ -n "<< parameters.plan_file >>" ]]; then
          PLAN_ARGS="$PLAN_ARGS -out=<< parameters.plan_file >>"
        fi
        export PLAN_ARGS
        terraform -chdir="$module_path" plan -input=false -no-color $PLAN_ARGS
```